### DISCUSSION TITLE
Wiki Formatting

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [avrdudes/avrdude#839] [Show and tell]
I forgot how painful forked wikis were..

*I still don't get why they aren't just cloned as part of the*
*repository itself so edits would just be normal pull requests.*

Anyways, I tried my hand at [`Home.md`].

[`Home.md`]: https://github.com/ElectronicsArchiver/avrdude/wiki

**Comment by mariusgreuel** [comment]
I think its all about preference. I like the existing format, compact and easy to maintain.

**Reply by ElectronicsArchiver** [reply]
I can't say I agree, while formatting is influenced by personal preference,
not / minimal usage is usually not benificial for readability. In fact projects
and their documentation usually become unmaintainable without it.
